app/src/main/res/values/strings.xml

```diff
@@ -690,30 +690,30 @@
     <string name="suggested_edits_unlock_notification_button">Yes, let’s go</string>
     <string name="suggested_edits_unlock_add_captions_dialog_title">You unlocked the "Add image captions" task</string>
     <string name="suggested_edits_unlock_add_captions_notification_title">You unlocked "Add image captions"</string>
-    <string name="suggested_edits_unlock_add_captions_dialog_message">Thanks for all your contributions! Since we think you like adding captions to images, we’ve compiled a list of images that are missing captions. Could you help us improve Wikipedia?</string>
-    <string name="suggested_edits_unlock_add_captions_notification_big_text">Suggested edits\n\nThanks for all your contributions. Since we think you like adding captions to images, we’ve compiled a list of images that are missing captions. Could you help us improve Wikipedia?</string>
+    <string name="suggested_edits_unlock_add_captions_dialog_message">Thanks for all your image caption edits! Since we think you like adding captions to images, we’ve compiled a list of images missing captions. Could you help us improving Wikipedia?</string>
```

Comment:
`Could you help us improving Wikipedia?` is incorrect grammar.

Reply:
Done.
https://phabricator.wikimedia.org/T225635#5288605